### PR TITLE
common.xml: add MAV_RESULT_CANCELED

### PR DESCRIPTION
**Pull request by amilcarlucas**

### CODE REVIEW

File: message_definitions/v1.0/common.xml [hunk 1]
```diff
@@ -1874,6 +1874,9 @@
       <entry value="5" name="MAV_RESULT_IN_PROGRESS">
         <description>Command is valid and is being executed. This will be followed by further progress updates, i.e. the component may send further COMMAND_ACK messages with result MAV_RESULT_IN_PROGRESS (at a rate decided by the implementation), and must terminate by sending a COMMAND_ACK message with final result of the operation. The COMMAND_ACK.progress field can be used to indicate the progress of the operation. There is no need for the sender to retry the command, but if done during execution, the component will return MAV_RESULT_IN_PROGRESS with an updated progress.</description>
       </entry>
+      <entry value="6" name="MAV_RESULT_CANCELLED">
+        <description>Command has been cancelled (as a result of receiving a COMMAND_CANCEL message).</description>
```
Comment (peterbarker) [inline message_definitions/v1.0/common.xml:1878]: We don't have `COMMAND_CANCEL`.

**Comment by tridge** [comment]
there is no COMMAND_CANCEL

**Comment by auturgy** [comment]
COMMAND_CANCEL is a wip message that hasn't been pulled down.  https://github.com/mavlink/mavlink/blob/041bf5c642d606550a0136ef6574f0295b7651bc/message_definitions/v1.0/common.xml#L4926
Suggest that this PR be ammended to include it, then we can discuss whether to bring it in.

**Comment by amilcarlucas** [comment]
Should I add the COMMAND_CANCEL (wip) message to this PR?

**Pull request closed without merging**